cmd/init.go

```diff
@@ -62,14 +72,29 @@ func (cli *CLI) init(opts Options) int {
 				}
 
 				if err != nil {
+					fmt.Fprint(cli.outStream, builder.String())
 					return fmt.Errorf("Failed to find a plugin; %w", err)
 				}
 
-				fmt.Fprintf(cli.outStream, "Plugin \"%s\" is already installed\n", pluginCfg.Name)
+				builder.WriteString(fmt.Sprintf("Plugin \"%s\" is already installed\n", pluginCfg.Name))
```

Comment:
This output is also buffered, so if the plugin is installed, all previous "already installed" messages are printed, but none are printed after that, so it's probably correct that it should never be printed.